src/main.js

```diff
@@ -187,8 +187,8 @@ const load = loader({
       monitor: monitor.prefix('handlers'),
       intree,
       reference,
-      jobQueueName: cfg.app.jobQueueName,
-      statusQueueName: cfg.app.statusQueueName,
```

Comment:
So I guess like this was the actual culprit, then? The queue names were being left at `undefined` which `consume` doesn't like.

Reply:
Exactly!